doc/source/installation.rst

```diff
@@ -2,11 +2,222 @@
 Installation
 ============
 
-At the command line::
+Using pip
+---------
+Use::
 
-    $ pip install python-redfish
+    sudo pip install python-redfish
 
-Or, if you have virtualenvwrapper installed::
+Pip will install :
+
+1. The library and all dependencies into your site-packages directory
+2. Redfish client master conf file into /etc/redfish_client.conf
+3. Data file (templates) into /usr/share/redfish-client/templates
+
```

Comment:
Solution discussed :
Use a config (and -c) through the CLI to overwrite path location of master conf file.
Rename current conf_file option name to --inventoryfile (-i).